Player/Scripts/Player.gd

```diff
@@ -24,16 +24,21 @@ enum {
 	ATTACK
 }
 
-var state = IDLE
-var flip = false
+var state:= IDLE
+var flip:= false
+export (bool) var attack 
 
-onready var sprite = $Body
-onready var anim = $AnimationPlayer
-onready var hold = $Hold
+onready var sprite := $Body
+onready var anim := $AnimationPlayer
+onready var hold := $Hold
 
-signal collecItem()
+signal collectItem
+signal attack(state)
 
 func _physics_process(_delta: float) -> void:
+	if Input.is_action_just_pressed("ui_end"):
+		global.reset(global.inventory)
+		emit_signal("collectItem")
```

Comment:
Reset inventory issue #8. Workaround for now is to manually reset using the end key.